sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -838,6 +845,20 @@ object SQLConf {
     .booleanConf
     .createWithDefault(false)
 
+  val ORC_META_CACHE_ENABLED = buildConf("spark.sql.orc.metadataCache.enabled")
+    .doc("To indicate if enable orc file meta cache, it is recommended to enabled " +
+      "this config in long-running process mode, such as Thrift Server, default is false")
+    .version("3.2.0")
```

Comment:
ditto. The same suggestion for the `config` namespace and the recommendation in the description.

Reply:
@dongjoon-hyun 
1. Now the change the 3 new config key start with `spark.sql.fileMetaCache.`, any better suggestions？
2. change the comments to `it is recommended to enabled  this config when multiple queries are performed on the same dataset